app/javascript/onboarding/Onboarding.jsx

```diff
@@ -70,18 +69,18 @@ export class Onboarding extends Component {
     const { currentSlide } = this.state;
     const { communityConfig } = this.props;
     return (
-      <FocusTrap>
-        <main
-          className="onboarding-body"
-          style={
-            communityConfig.communityBackground && {
-              backgroundImage: `url(${communityConfig.communityBackground})`,
-            }
+      <main
```

Comment:
Related to #1153, but we should mark a TODO here to have the skiplink work here.